dipy/viz/tests/test_actors.py

```diff
@@ -103,12 +103,15 @@ def test_slicer():
                                       value_range=(0., 1.))
     renderer.clear()
     slicer_lut = actor.slicer(data, lookup_colormap=lut)
-
     slicer_lut.display(10, None, None)
     slicer_lut.display(None, 10, None)
     slicer_lut.display(None, None, 10)
+    slicer_lut.opacity(0.5)
 
     slicer_lut2 = slicer_lut.copy()
+    npt.assert_equal(slicer_lut2.GetOpacity(),0.5);
```

Comment:
No semicolon. Consider using a pep8 linter with your editor to catch little things like this.

Reply:
done